### ISSUE TITLE
Select tab by number

### ISSUE THREAD

**Issue by ianb** [mozilla/firefox-voice#1082]
We have next/previous/first/last tabs (implemented in #1078). We could also support numbers, like "go to tab 23" or "go to second tab".

It would probably be good to extend the `smallNumber` type in `entityTypes.js` (in a new type), and include cardinals (first, second, third, etc).

**Comment by irvin** [comment]
we tried to implement this intent in the local workshop, and bumped into difficulties that commands like "go to the third tab" or "go to the four tab" often matched by `find.find`. I think we need to limit find.find in a more restrictive way or make it fallback of other intents.

**Comment by ianb** [comment]
Did you add a new type, like `[tabNumber:numberOrCardinal]` ? If the slot is typed it should bind more actively than the wildcard that find.find looks for.

You can test this by creating an example and using `npm run jest` to test the example. It will give an error if your example is instead interpreted by `find.find`

For creating `numberOrCardinal` you'd add something like this to entityTypes.js:

```js
export const entityTypes = convertEntities({
  ...
  numberOrCardinal: [
    "first", "second", "third", ...
    ...smallNumber
  ]
});
```

(This is assuming you split out `smallNumber` so it can be reused. And maybe add 10-20 to it.)

**Comment by sabina-rohman** [comment]
Hi, @ianb  I'm an outreachy applicant May 2020. Can I please work on this issue?

**Comment by richardgreg** [comment]
Hi @ianb. I am an Outreachy Applicant. Pls assign me to this issue.

**Comment by PrastutiUpadhaya** [comment]
Hi, can I work on this issue? I am an Outreachy applicant.

**Comment by ianb** [comment]
I'm going to assign this to @sabina-rohman; Sabina, if you aren't able to complete this please comment so we can open it up to other people.

**Comment by sabina-rohman** [comment]
@ianb Thanks. I will update here.

**Comment by ianb** [comment]
@sabina-rohman Are you still working on this? If it's not working out I'd like to open this back up for other contributors

**Comment by sabina-rohman** [comment]
@ianb yes, I am working on it right now. I am following your suggestion and adding a numberOrCardinal to the entityTypes and have a go. Will keep you updated :)

**Comment by sabina-rohman** [comment]
@ianb I created PR for this ticket : https://github.com/mozilla/firefox-voice/pull/1199
Can you please review? Not sure what the error is about. I ran the tests locally and they were green.

**Comment by creatorpiyush** [comment]
Can I take this issue if no one is working on it?

**Comment by PascalUlor** [comment]
Hi I'll like to work on this if it's still available
